python/cudnn/sparse_attention/fwd/sm100_gqa/gqa_prefill_bf16_tcgen05_sm100.py

```
@@ -0,0 +1,943 @@
+        # in-bounds ``tile_start=0`` for an invalid/absent entry (see its
+        # docstring), so an inactive slot's TMA loads block 0 of this
+        # CTA's KV head -- wasted bandwidth, never touched by masked-out
+        # compute, but always present so the mbarrier phase counters the
+        # consumer (all 4 warps) waits on stay in lock-step with the
+        # producer (warp 0) regardless of this slot's validity.
+        for j in range(topk_max):
+            j_i32 = cutlass.Int32(j)
+            active = j_i32 < n_entries
+            raw_entry = cutlass.Int32(idx_v[t_q_rep, kv_head, j_i32])
+            entry = raw_entry if active else cutlass.Int32(-1)
+            tile_start, num_valid, is_valid = resolve_entry_window(entry, TILE_N, kv_bound)
```

Ruff (0.16.3)

[warning] Unpacked variable `num_valid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)